config/manager/manager.yaml

```diff
@@ -39,47 +39,49 @@ spec:
         seccompProfile:
           type: RuntimeDefault
       containers:
-      - command:
-          - /usr/bin/cert-manager-operator
-        args:
-          - start
-          - "--v=$(OPERATOR_LOG_LEVEL)"
-          - "--trusted-ca-configmap=$(TRUSTED_CA_CONFIGMAP_NAME)"
-        env:
-          - name: WATCH_NAMESPACE
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.annotations['olm.targetNamespaces']
-          - name: POD_NAME
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.name
-          - name: OPERATOR_NAME
-            value: cert-manager-operator
-          - name: RELATED_IMAGE_CERT_MANAGER_WEBHOOK
-            value: quay.io/jetstack/cert-manager-webhook:v1.11.0
-          - name: RELATED_IMAGE_CERT_MANAGER_CA_INJECTOR
-            value: quay.io/jetstack/cert-manager-cainjector:v1.11.0
-          - name: RELATED_IMAGE_CERT_MANAGER_CONTROLLER
-            value: quay.io/jetstack/cert-manager-controller:v1.11.0
-          - name: OPERAND_IMAGE_VERSION
-            value: 1.11.0
-          - name: OPERATOR_IMAGE_VERSION
-            value: 1.11.0
-          - name: OPERATOR_LOG_LEVEL
-            value: "2"
-          - name: TRUSTED_CA_CONFIGMAP_NAME
-        image: controller:latest
-        imagePullPolicy: Always
-        name: cert-manager-operator
-        securityContext:
-          allowPrivilegeEscalation: false
-          capabilities:
-            drop:
-              - "ALL"
-          privileged: false
-          runAsNonRoot: true
-          seccompProfile:
-            type: "RuntimeDefault"
+        - command:
```

Comment:
This diff seems to be larger than usual, not sure why. Though you'd only change a couple of lines and the whole block as-is.

Reply:
Seems that the indentation was incorrect

Reply:
oh cool